pibooth/booth.py

```diff
@@ -117,6 +117,7 @@ def __init__(self, config):
         self.previous_picture = None
         self.previous_animated = None
         self.previous_picture_file = None
+        self.second_previous_picture = None
```

Comment:
Could be an attribute of the `picture_plugin` as it is intended for its work

Reply:
Good idea, done!